### PR TITLE
Cleaning up deprecated files 

### PR DESCRIPTION
**Pull request by phiden**
See issue: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/4422

### CODE REVIEW

**Review by robbiethegeek-usds — APPROVED** [review]
@annekainicUSDS can you give a quick review, once she does merge away @phiden

**Comment by ayaleloehr** [comment]
Anne is on vacation for the next week, fyi

**Comment by ayaleloehr** [comment]
@phiden Can this be merged or find a reviewer other than Anne since she is out?

**Comment by phiden** [comment]
@ayaleloehr I'll ask for another reviewer -- I know robbie wanted another set of eyes

File: content/includes/L2-section-landing.html
Comment (jkassemi) [inline content/includes/L2-section-landing.html]: 😄 

Can you also remove the `csp-inline-path*` rules that were added to `src/sass/_shame.scss`?

**Review by jkassemi — APPROVED** [review]
👍 - minor request in notes

**Comment by phiden** [comment]
@jkassemi rules removed, if you want to take a look!

**Pull request merged**
